docs/end-to-end-test.md

```diff
@@ -0,0 +1,166 @@
+# Overview
+
+This test aims to simulate the SAPS processing pipeline using test scripts.
+
+# Test flow
+
+![Test flow](img/end-to-end-test-flow.png)
+
+1. User submits a task (new processing) to the Dashboard
+2. Dashboard forwards the processing request to the Dispatcher
+3. Dispatcher adds new processing information to the Catalog
+4. Scheduler retrieves the new information entered for processing
+5. Scheduler submits the task to a Job Scheduler Service that will process the scripts
```

Comment:
Wouldn't it be a Job Execution Service?

Reply:
Done, I made a mistake regarding the service

Reply:
I will update the image too ...

Reply:
Image updated